src/workerd/api/filesystem.c++

```diff
@@ -1965,7 +1965,8 @@ jsg::JsValue createUVException(jsg::Lock& js,
 
 namespace {
 constexpr bool isValidFileName(kj::StringPtr name) {
-  return name.size() > 0 && name != "."_kj && name != ".."_kj && name.find("/"_kj) == kj::none;
+  return name.size() > 0 && name != "."_kj && name != ".."_kj && name.find("/"_kj) == kj::none &&
```

Comment:
Can you add a comment to this function explaining what a valid file name consist of?

Reply:
I'm going to be coming back to this function and doing some perf optimization on it. I'll add the comment then.